src/apps/frontend/components/form-container/form-header.component.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropsWithChildren } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+interface FormHeaderProps {
+  navigateBackwardURL?: string;
+}
+
+const FormHeader: React.FC<PropsWithChildren<FormHeaderProps>> = ({
+  children,
+  navigateBackwardURL,
+}) => {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      {navigateBackwardURL
```

Comment:
This seems poor abstraction, what is someone wants to add a different CTA or something else